django_elasticsearch_dsl/documents.py

```diff
@@ -191,10 +191,10 @@ def update(self, thing, refresh=None, action='index', parallel=False, **kwargs):
         """
         Update each document in ES for a model, iterable of models or queryset
         """
-        if refresh is True or (
-            refresh is None and self.django.auto_refresh
-        ):
+        if refresh is None and self.django.auto_refresh:
             kwargs['refresh'] = True
+        elif refresh is not None:
+            kwargs['refresh'] = refresh
```

Comment:
I think if `refresh` parameter is passed, it should be provided the highest priority regardless what is mentioned in the document. What do you think? @chosak